conf/ocs_version/ocs-4.7.yaml

```diff
@@ -0,0 +1,10 @@
+---
+DEPLOYMENT:
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-stable-4.7"
+  default_latest_tag: 'latest-stable-4.7'
```

Comment:
I would go with latest-4.7 for now, it can make some time till the build will pass the acceptance and we can change it once we have it

Reply:
Would I need to change the `ocs_csv_channel` as well if we aren't going with latest-stable or does that remain `stable-4.7`?

Reply:
csv channel is always in that format stable-X.Y ;) so no need to change.